### PR TITLE
Add ryejs@0.1.2 w/ npm-autoupdate

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern @cdnjs/intern2, would you please help me checkout the pr?
Thank you.💦
The PR is for #9185.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/ryejs/rye
- Official website (optional, not the repository): http://ryejs.com
- NPM package url (optional): https://www.npmjs.com/package/ryejs
- GitHub / Bitbucket popularity (required):
  - Count of stars: 259
  - Count of watchers: 19
  - Count of forks: 18
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
http://ryejs.com/dist/rye-0.1.0.js
rye-0.1.0.js is a collection of all `.js`s in `/lib`, so we should host them all.
since there is only one version, you can ask the authors if they can also provide `rye-0.1.0.min.js` on GitHub.

File: ajax/libs/rye/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "rye",
+  "filename": "rye.min.js",
+  "version": "0.0.1",
```
Comment (maruilian11) [inline ajax/libs/rye/package.json]: the latest version is 0.1.0, you can add it by git log tracing, and ask the authors if they can keep adding git tags.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2,
dear all, the author had published the library on npm :smile: , so that I use npm autoupdate instead of git autoupdate.
Furthermore, I add the old version published on homepage.
Thank you all!

**Comment by maruilian11** [comment]
@Piicksarn please confirm the license with the authors.
https://github.com/ryejs/rye/blob/master/LICENSE

**Comment by Piicksarn** [comment]
@pvnr0082t, 
The files we needed in version 0.01 are not commited to the git repo.
They should create by manually running CoffeeScript command.
Is that still fine to include, since the files are not officially commited to the git repo?

**Comment by pvnr0082t** [comment]
@Piicksarn Okay, I think maybe we could leave it alone. LGTM

**Comment by PeterDaveHello** [comment]
ping @cdnjs/intern2

**Comment by x09326** [comment]
@Piicksarn Please rebase to the latest master branch, thanks.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2,
dear all, I have it rebase to master and need ur help for reviewing, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by maruilian11** [comment]
have you opened the issue about the license?

File: ajax/libs/ryejs/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "DOM",
+    "browser",
+    "modern"
+  ],
+  "homepage": "http://ryejs.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ryejs/rye.git"
+  },
+  "license": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/ryejs/package.json]: license wrong ... @x09326 @pvnr0082t @Piicksarn

Comment (x09326) [inline ajax/libs/ryejs/package.json]: The license in github is different from npm.
@Piicksarn You need to confirm that with the author.

**Comment by PeterDaveHello** [comment]
Issue number not mention in the first comment.

**Comment by Piicksarn** [comment]
@PeterDaveHello, I did! :disappointed_relieved: 
And the License problem had be mentioned in following msg of this issue :point_right: https://github.com/ryejs/rye/issues/34

**Comment by PeterDaveHello** [comment]
I mean #9185

**Comment by WesleydeSouza** [comment]
@PeterDaveHello @Piicksarn license is ISC as of [a790a09a](https://github.com/ryejs/rye/commit/a790a09a77a518de121720e41e245ae220403f61).

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2,
Dear all, I have its' license modified and need ur help for reviewing.
Thank you all!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@Piicksarn you should mention the reason why you manually added v0.1.0 in commit message and PR comment, we talk about this problem several times.

**Comment by WesleydeSouza** [comment]
@Piicksarn I guess you could remove v0.1.0 from this PR, we're going to update the website to reflect the 0.1.2 version soon.

**Comment by extend1994** [comment]
Hi @WesleydeSouza 
Sorry for no response for so long. I am going to add this library now!
So we don't need v0.1.0, right?
Thanks!

**Comment by WesleydeSouza** [comment]
Yes.

**Pull request closed without merging**